Tools/clinic/libclinic/function.py

```diff
@@ -167,6 +167,19 @@ def methoddef_flags(self) -> str | None:
             flags.append('METH_COEXIST')
         return '|'.join(flags)
 
+    @property
+    def docstring_line_width(self) -> int:
+        """PEP 8 requires that docstrings are limited to 72 characters:
```

Comment:
PEP 8 is perhaps not relevant here, as this is not Python code.

But `pydoc` adds indentation 4 characters for global functions and classes and 8 characters for methods. To make the total length not exceeding 80 columns, the limit should be 76 for module-level functions, `__new__` and `__init__`, and 72 for other methods. I am sorry for misleading you.

I think that it is better to refer pydoc than PEP 8 here.

Reply:
> ... the limits should be 76 and 72.
> 
> PEP 8 limits docstring line length in Python code, but this is not Python code.

True, but docstrings are a visible artefact from Python, so I'm not opposed to 72 characters.

I will update to 76 & 72.